final_task/rss_reader/pdf_and_html_converting.py

```diff
@@ -0,0 +1,130 @@
+import urllib
+from fpdf import FPDF
+import os
+import shutil
+
+
+def drawing_image(file_name):
+    from PIL import Image
+    from PIL import ImageFont
+    from PIL import ImageDraw
+
+    img = Image.new("RGB", (480, 360))
+    draw = ImageDraw.Draw(img)
+    font = ImageFont.truetype('arial', 60)
+    draw.text((120, 120), "No image", (255, 255, 255), font=font)
+    img.save('{0}'.format(file_name))
+
+
+def checking_path(path):
+    if path.endswith('/') or path.endswith("\\"):
+        return 'Imag\\'
+    else:
+        return '\\Imag\\'
```

Comment:
Checkout `os.path` lib  for working with filesystem because right now this would work very wrong on Linux